stestr/commands/load.py

```diff
@@ -116,6 +129,16 @@ def make_tests():
         inserter = repo.get_inserter(partial=partial, run_id=run_id)
     if subunit_out:
         output_result, summary_result = output.make_result(inserter.get_id)
+    if pretty_out:
+        outcomes = testtools.StreamToDict(
+            functools.partial(subunit_trace.show_outcome, sys.stdout,
+                              enable_color=color))
+        summary_result = testtools.StreamSummary()
+        output_result = testtools.CopyStreamResult([outcomes, summary_result])
+        output_result = testtools.StreamResultRouter(output_result)
+        cat = subunit.test_results.CatFiles(sys.stdout)
+        output_result.add_rule(cat, 'test_id', test_id=None)
+        start_time = datetime.datetime.utcnow()
```

Comment:
Yep, start time isn't defined in the case pretty_out is false. So when L157 is run there is no start time defined. I should have put start time definition out one indent level and to L144. I'll respin it right now